doc/COOKBOOK-FIXERS.md

```diff
@@ -444,18 +444,14 @@ The review usually flows like this:
 caveats. Usually, the person building a fixer is blind about some
 behavior mistakes of fixers. Expect to write few more tests to cater for
 the reviews.
-2. People will discuss the relevance of your fixer. If it is
-something that goes along with Symfony style standards, or PSR-1/PSR-2
-standards, they will ask you to move from Symfony/CS/Fixers/Contrib to
-Symfony/CS/Fixers/{Symfony, PSR2, etc}.
```

Comment:
this is partially still true. ppl will discuss and ask to add it to ruleset instead of level/namespace